backend/LexBoxApi/Services/ProjectService.cs

```diff
@@ -89,6 +89,33 @@ public async Task FinishReset(string code, Stream? zipFile = null)
         return lastCommitFromHg;
     }
 
+    public async Task<int?> GetLexEntryCount(string projectCode)
+    {
+        var project = await dbContext.Projects.Include(p => p.FlexProjectMetadata).FirstOrDefaultAsync(p => p.Code == projectCode);
+        if (project?.MigrationStatus is not ProjectMigrationStatus.Migrated) return null;
+        if (project?.Type is not ProjectType.FLEx) return null;
+        if (project?.FlexProjectMetadata?.LexEntryCount is int c) return c;
+        return await UpdateLexEntryCount(projectCode);
```

Comment:
calling this here will execute 2 more queries, one to query the project again, and one to update it in the db. Can we simplify that?

Reply:
I thought about passing `project` instead of `projectCode`; I'll double-check all the places this gets used and see if that's viable. It probably is, though possibly the ProjectController update API method will go from one line to "fetch project, then pass the project to UpdateLexEntryCount" since I think that's the one code path that doesn't pass through GetLexEntryCount first.